JitsiConferenceEventManager.js

```diff
@@ -529,38 +530,64 @@ JitsiConferenceEventManager.prototype.setupRTCListeners = function() {
         });
 };
 
+/**
+ * Removes event listeners related to conference.xmpp
+ */
+JitsiConferenceEventManager.prototype.removeXMPPListeners = function() {
+    const conference = this.conference;
```

Comment:
Why do you need to use this intermediary variable? It's for convenience?

Reply:
Yes, just for convenience. Basically I've copied from setupXMPPListeners.